client/src/components/navbar/Navbar.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function Navbar() {
+  return (
+    <nav id='Navbar'>
+      <img src='https://logos-world.net/wp-content/uploads/2021/10/Sonic-Logo-700x394.png' alt='Sonic Logo' />
```

Comment:
this url would be better placed as the value for the `background` rule in a css selector